pkgs/by-name/vz/vzic/package.nix

```
@@ -0,0 +1,51 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  pkg-config,
+  glib,
+  tzdata,
+  libical,
+}:
+stdenv.mkDerivation rec {
+  pname = "vzic";
+  version = "2024a";
```

This should denote some unique-ish identifier of upstream's version. As-is, this only says which `tzdata` directory will be used for building. That's insufficient, because it's present in multiple commits since its introduction.

```suggestion
  version = "0-unstable-2024-03-09";
```

I think if you want the information to be present in the name, maybe it should be added as a postfix to the `pname`? I.e. `vzic-tzdata2024a`. version is the wrong place, *maybe* unless you pin `src.rev` to the commit that introduced tzdata2024a: https://github.com/libical/vzic/commit/7ff0ce05c32ad3218cd12f71e6194dd1f7add299

---

I prefer to pin the rev to that commit. Putting the tzdata version in pname seems to be weird.

---

I think you should use what @OPNA2608 advised here.